plugin.video.haystacktv/addon.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<addon id="plugin.video.haystacktv" version="1.0.0" name="Haystack.TV" provider-name="Lunatixz">
+    <requires>
+        <import addon="xbmc.python" version="2.25.0"/>
+        <import addon="script.module.simplecache" version="1.0.0"/>
+        <import addon="script.module.beautifulsoup4" version="4.3.2"/>
+        <import addon="script.module.urlresolver" version="0.0.1"/>
+        <import addon="plugin.video.youtube" version="5.1.8" optional="true"/>
+        <import addon="plugin.video.vimeo" version="4.1.4" optional="true"/>
+    </requires>
+    <extension point="xbmc.python.pluginsource" library="default.py">
+        <provides>video</provides>
+    </extension>
+    <extension point="xbmc.addon.metadata">
+        <summary lang="en_GB"></summary>
+        <description lang="en_GB">Haystack TV is your personal headline news channel. We learn about your interests in order to surface the news that matters to you! Watch it on Kodi today...</description>
+        <platform>all</platform>
+        <license>GNU GENERAL PUBLIC LICENSE. Version 3, June 2007</license>
+        <source>https://github.com/Lunatixz/XBMC_Addons/tree/master/plugin.video.haystacktv</source>
+        <website>http://www.haystack.tv/</website>
+        <forum>https://forum.kodi.tv/showthread.php?tid=316485</forum>
+        <news></news>
+        <assets>
+            <icon>icon.png</icon>
```

Comment:
As this is a new submission and you are targeting krypton as a base version you should probably respect the new addon structure: move both files to the resources folder. See: http://kodi.wiki/view/Add-on_structure